### PR TITLE
Add Oracle Linux 9 and Ubuntu Jammy in qa_pxc_57_80_test Job

### PR DESCRIPTION
**Pull request by kaushikpuneet07**

### CODE REVIEW

File: pxc/jenkins/qa_pxc_57_80_test-pipeline.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@
     pipeline-scm:
       scm:
         - git:
-            url: https://github.com/Percona-Lab/jenkins-pipelines.git
+            url: https://github.com/kaushikpuneet07/jenkins-pipelines.git
```
Comment (eleo007) [inline pxc/jenkins/qa_pxc_57_80_test-pipeline.yml:10]: revert repo

**Comment by eleo007** [comment]
Hi,
Could you share successful run with changes?

**Comment by kaushikpuneet07** [comment]
> Hi, Could you share successful run with changes?

https://pxc.cd.percona.com/job/qa_pxc_57_80_test-param/33/

File: pxc/jenkins/qa_pxc_57_80_test-pipeline.yml (hunk 1 above)
Comment (kaushikpuneet07) [reply pxc/jenkins/qa_pxc_57_80_test-pipeline.yml]: Done

**Review by eleo007 — APPROVED** [review]

**Pull request merged**